torch/ao/quantization/fx/prepare.py

```diff
@@ -1316,6 +1322,23 @@ def insert_observers_for_model(
 
     return results_node
 
+def _validate_fixed_qparams_qconfigs(model: GraphModule, qconfig_mapping: QConfigMapping):
+    """
+    Validate whether the correct observers are configured for fixed qparams ops in the model, if any.
+    """
+    # TODO: handle fp16 qconfigs properly
+    allowed_qconfigs = {
+        float16_static_qconfig,
+    }
+    for node in model.graph.nodes:
+        if node.target in _FIXED_QPARAMS_OP_TO_OBSERVER:
+            qconfig = get_object_type_qconfig(qconfig_mapping, node.target, qconfig_mapping.global_qconfig)
```

Comment:
I feel we should get the qconfig for this node from the generated `qconfig_map`, since user may use different ways of configuration to set the qconfig for these nodes

Reply:
in L1517 of this file:
```
qconfig_map = generate_qconfig_map(model, modules, model.graph, qconfig_mapping, node_name_to_scope)
```
we should be using this one I think, also maybe makes sense to rename it since now we have `qconfig_mapping`

Reply:
I see, OK. I feel we really need to rename it now :)

Reply:
I see, OK. Yes I feel we really need to rename it now :)